examples/wc-wallet-react/src/index.tsx

```diff
@@ -9,17 +9,26 @@ import {
   DEFAULT_METHODS,
 } from './constants'
 import App from './App'
-import { WalletConnectWalletProvider } from '@cityofzion/wallet-connect-sdk-wallet-react'
+import {
+  DEFAULT_AUTO_ACCEPT_METHODS,
+  TOptions,
+  WalletConnectWalletProvider,
+} from '@cityofzion/wallet-connect-sdk-wallet-react'
 import { AccountContextProvider } from './context/AccountContext'
 
-const wcOptions = {
+const wcOptions: TOptions = {
```

Comment:
I think `TOptions` is too generic, it would be better to have a more descritive name, the context of this options, and we are not using the `T` prefix anywhere, right? I would rename to `WcInitOptions` or something like this.

Reply:
My bad, we are actually using T prefix, but I never saw it exposed to the user, idk...